src/component/Slider/Slider.js

```diff
@@ -0,0 +1,59 @@
+import './Slider.scss';
+
+function Slider() {
+  return (
+    <div className="container-beige-title__navbox">
+      <div className="container-beige-title__nav">
+        {SLIDER_NAV_DATA.map(navdata => {
+          return (
+            <div key={navdata.id} className="container-beige-title__nav-imgbox">
+              <img alt={navdata.alt} src={navdata.src} />
+              <label>{navdata.name}</label>
+            </div>
+          );
+        })}
```

Comment:
map 메서드 안에서도 구조분해할당 적용할 수 있습니다!
```suggestion
        {SLIDER_NAV_DATA.map(navdata => {
          const { id, alt, src, name } = navdata;
          
          return (
            <div key={id} className="container-beige-title__nav-imgbox">
              <img alt={alt} src={src} />
              <label>{name}</label>
            </div>
          );
        })}
```

Reply:
넵! 적용하겠습니다 :)